### ISSUE TITLE
Pre-trained models results for different epochs 

### ISSUE THREAD

**Issue by moabarar** [google-research/simclr#119]
Hi,

Thanks for sharing your code. I have couple of questions regarding the provided command and the results of SimCLRv2 trained on various epochs.

[1] In the ReadMe file, it is said that running the linear evaluation protocol yields accuracy around 64.5%. This accuracy regarding training ResNet50 using SimCLRv2 for 100 epochs - right ?

[2] Are there any results available for linear evaluation when training SimCLRv2 on different epochs (e.g., 200, 400). In the paper the only results reported are for training the network for 800 epochs. Something similar to the results in Table B.1 in [SimCLRv1 paper](https://arxiv.org/pdf/2002.05709.pdf).  

Thanks in advance

**Comment by chentingpc** [comment]
[1] That number was for SimCLRv1, not for v2.
[2] We didn't report those numbers for SimCLRv2. But if you're interested, please refer to Table D.3 of this [paper](https://arxiv.org/pdf/2011.02803.pdf) for numbers of different epochs and batch sizes, which closely follows SimCLRv2 changes but without momentum encoder.

**Comment by moabarar** [comment]
Thanks for the reply!
